app/models/invite.rb

```diff
@@ -0,0 +1,20 @@
+class Invite < ActiveRecord::Base
+  # include the TokenGenerator extension
+  include TokenGenerator
+  belongs_to :website
```

Comment:
so add this here:
`belongs_to :inviter, foreign_key: :invited_by_id, class_name: User`